content/_data/changelogs/weekly.yml

```diff
@@ -12362,6 +12363,52 @@
   # pull: 5671 (PR title: Mark Resource Domain Recommendation monitor as security)
   # pull: 5676 (PR title: Bump mockito-core from 3.11.2 to 3.12.1)
 
+  - version: '2.309'
+    date: 2021-08-30
+    changes:
+      - type: rfe
+        category: developer
+        pull: 5682
+        authors:
+          - dependabot[bot]
+        pr_title: Bump jna from 5.8.0 to 5.9.0
+        references:
+          - pull: 5682
+          - url: https://github.com/java-native-access/jna/blob/master/CHANGES.md#release-590
+            title: JNA 5.9.0 changelog
+        message: |-
+          Developer: Bump Java Native Access (jna) from 5.8.0 to 5.9.0.
+      - type: rfe
+        category: internal
+        pull: 5687
+        authors:
+          - basil
+        pr_title: Mark `PluginFirstClassLoader`, `MaskingClassLoader`, and `AntClassLoader`
+          as parallel-capable
+        message: |-
+          Internal: <code>AntClassLoader</code> (and its subclass <code>PluginFirstClassLoader</code>) and <code>MaskingClassLoader</code> register themselves as parallel-capable.
+      - type: rfe
+        category: rfe
+        pull: 5542
+        authors:
+          - MadsJakobsen
+        pr_title: Add `ABORTED` threshold to ReverseBuildTrigger
+        message: |-
+          Add <code>ABORTED</code> threshold to <code>ReverseBuildTrigger</code>
```

Comment:
```suggestion
          Add <code>ABORTED</code> threshold to <code>ReverseBuildTrigger</code>.
```

Reply:
Best to copy edit in the PR, otherwise it’s gets overwritten on merge.

once the weekly is released final cleanup can be done here

it would be good to create issues for the automated changelog